ibm_catalog.json

```diff
@@ -668,6 +666,14 @@
                             "key": "region",
                             "required": true
                         },
+                        {
```

Comment:
@ocofaigh this is supposed to be removed right? I think it got back in via a merge resolve

Reply:
yep, we no longer need to explicitly specify required variables - catalog now marks them as required if they dont have a default value in terraform